Makefile

```diff
@@ -1,4 +1,18 @@
-.PHONY: all install uninstall reinstall clean doc test
+.PHONY: all clean doc test bench
 
-all install uninstall reinstall clean doc test:
-	$(MAKE) -C src $@
+all:
+	dune build -p pprint
```

Comment:
What's the reason for this `-p pprint`?

Reply:
It was just the same as in the opam file. Though @trefis just told me that this will disable warnings during development so I will remove `-p pprint` :)